---
title: "Gene Expression: A current and prospective summary"
author: "Gustavo Magaña López"
email: "gustavo.magana-lopez@u-psud.fr"
date: "30/12/2020"
#header-includes:
 #  - \usepackage{wrapfig}
output:
  pdf_document:
    includes:
      in_header: "preamble.tex" 
    toc: true
    #toc_depth: 1
    citation_package: biblatex
    #fig_width: 7
    #fig_height: 6
    #fig_caption: true
bibliography: "references.bib"
nocite: '@*'
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```
\newpage
# The cell's congitive problem

The genetic alphabet is a simple one, consisting of just 4 letters, but it makes
up for all lifeforms known to humanity. Genes are at the centre of 
human development and differentiation. Furthermore, they play a central role in 
numerous diseases. (_remember to check_) **The latest research focused both
on diagnosis therapy, notably in oncology** is driven by the refinement of
molecular biology techniques and the advancement of [OMICS](https://en.wikipedia.org/wiki/Omics). 

The _central dogma_ states that information flows from DNA to RNA and finally to 
protein via _transcription_ and _translation_. These are the mechanisms by which
the genotype of an individual is converted to its phenotype. 
Organisms such as _E. coli_ can produce about 4,500 different proteins. Each one 
of our cells contains up to 200,000 genes. In order to maintain homoeostasis, 
cells must react and adapt to diverse stimuli. Hence, there must be a defined 
control mechanism permitting efficient and appropriate expression of genes, in
accordance to the organism's current state and its environment.

So far proteins and genes have been mentioned, but what is a gene? A gene is a 
region of the genome of the organism. 
Constituted by two parts, the _coding region_ which specifies the amino acids
that will make up the protein, and the _promoter region_, 
which allows the gene to be switched "on" or "off".

faire l'introduction après.



The regulation of gene expression is thus, the dynamic link between the quasi
static information contained in the genome and the vast array of internal metabolic
states derived from external stimuli. 

### Extending to multicellular eukaryotes

Talk about cellular differentiation, leading to the formation of tissues, organs
and a full coordinated and integrated organism.

\newpage
# How to regulate the response 

### Gene loss 

A possible method of gene control could involve the genes for all proteins 
being present in the fertilised egg. Then, during development, the selective 
loss of genes which are not required according to the cell/tissue type 
(in the case of multicellular eukaryotes) would determine gene expression. 
Under this hypothesis cells would only have _housekeeping genes_ and their 
characteristic genes. 
This would eliminate the need for additional means of gene regulation as 
regards cell differentiation. 
If this were the general case, the presence/absence of mRNA coding for specific
proteins in the cytoplasm would be matched in the DNA contents of the nucleus.

A variety of evidence suggests, however, that **selective DNA loss is not a 
general mechanism of gene control**. This is based on chromosomal, functional,
and molecular studies. It has to be noted that there are some specific cases 
in which this mechanism is used (embryonic development of some nematodes, 
crustaceans, and insects). This will not be discussed as it goes beyond the 
scope of this summary. 

### Gene amplification 

Having discarded _gene loss_ as a general means of gene control, we must examine
the role _gene amplification_ in the regulation of gene expression. Under this
hypothesis, selective amplification of genes would account for the observed 
expression profile (mRNA and protein concentrations), keeping normal 
transcription and translation rates. Unlike the _gene loss_ or _gene-deletion_
models, this proposed mechanism is not incompatible with renowned 
experiments such as the cloning of Dolly[^1], achieved via nuclear transfer from
an adult somatic cell into an unfertilised oocyte. 

As with _gene loss_, the identity of the chromosomal complement in individual 
cell types provides strong evidence against the occurrence of large-scale _DNA
amplification_. Furthermore, the evidence coming from molecular studies such as
_Southern blotting_ sustains the premise that across different tissues, 
the number of copies of each gene stays constant. As mentioned previously, this
discards _gene amplification_ as the general regulation mechanism, although 
it happens under certain normal and pathological conditions. Two cases are the
amplification of ribosomal DNA during embryonic development of multicellular 
organisms and oncogene amplification in some cancer cells.


[^1]: Add the reference 


Both _gene amplification_ and _gene loss_ are possible mechanisms of gene regulation,
but as previously stated, they seldom happen in specific contexts. 
In addition, the previously stated _gene regulation hypotheses_ would 
only account for tissue-specific cellular differentiation in 
multicellular eukaryotes, having no validity in unicellular organisms. 
An example of gene expression as a response
to external stimuli is the case of the _lac operon_ of _E. coli_. 
In a study of a wide range of yeast proteins and mARNs, these two generally 
had a strong positive correlation. These were measured via two-dimensional
gel electrophoresis and a gene chip methods respectively.

From this brief discussion and the positive correlation of
mRNA and protein concentrations, we can conclude that gene regulation must 
predominantly operate to produce different amounts of mRNA from a select
subset of all the genes ubiquitously present. **The key to gene expression must
lie therefore in the process of _transcription_**.


## Transcription

Some details about transcription differ between eukaryotes and prokaryotes.
These will be treated in their respective sections, but in general
transcription involves the following steps: access of transcription apparatus 
to DNA, recognition of promoter sequences, initiation of RNA synthesis, 
elongation of RNA, and termination. 
Any of these individual steps may be regulated, but in practice, 
regulation of certain steps is much more common than the regulation of others.


## Procaryotic Transcription

The process of gene expression or gene regulation is mainly mediated via transcription.
We must therefore understand the bases of this process to discuss gene expression.
Bacteria can regulate gene expression via DNA-binding proteins. 
These can be categorised either as **repressors** or **activators**, which can be 
subject to allosteric regulation themselves. 
The core transcription process in prokaryotes is explained by the **operon model**. 

An _operon_ is a segment of DNA that consists of a regulatory section and 
a coding section.
The _regulatory section_ consists of a sequence of DNA with control sites known 
as the _promoter_ and the _operator_ sites.
The _coding section_ contains the gene(s) that code for the polypeptide(s) that
will make up the protein(s). 
Additionally, a distinction must be made between two types of regulators. 
Those located on the same DNA fragment which they regulated are called 
_**cis-elements**_. On the other hand, proteins that bind to specific sites 
of the sequence either to enhance or repress the expression of the gene 
in question are called _**trans-elements**_ for they are not necessarily
encoded on the same DNA region. These proteins are also called 
_transcription factors_.

![Simplified schematic representation of the _Lac Operon_](figures/my_lac)

The classic example of gene regulation in prokaryotes is the _Lac Operon_. 
When glucose concentration is high, RNA polymerase binds to the promoter region 
(1) on the Lac operon and transcribes the mRNA that codes for the repressor protein.
Once formed, the repressor protein binds to the operator (5) region and blocks the
RNA polymerase from transcribing the mRNA that codes for $\beta$-galactosidase,
responsible for catalysing the breakdown of lactose into glucose and galactose.

This occurs because when glucose is available, specially so in high concentrations,
the cell does not need to use lactose (or other sugars for that matter) as
an energy source. 
Thus, there is no need to to express the gene for $\beta$-galactosidase.

However, when the glucose concentration is low and lactose is available, 
lactose can transform into allolactose, which binds to the repressor protein 
allosterically. 
This inactivates the repressor and causes it to detach from the operator.
At low glucose concentrations, the concentration of cAMP rises and cAMP 
binds to and activates the _catabolite activator protein_ **CAP**.
CAP binds to a special site (3) on the Lac operon, which activates the promoter
for $\beta$-galactosidase. RNA polymerase can now bind to and transcribe the 
mRNA for $\beta$-galactosidase. 

Under the light of "_cognitive problem of the cell_", the regulation of the 
_Lac Operon_ has a clear functional role as a response to the availability 
of different sugars in E. coli's environment. In order to maintain its normal
activity and ultimately live, the cell needs to obtain energy from ATP. 
In E. coli this is mainly achieved via glycolysis. 
If glucose is unavailable, the cell must obtain energy from other sugars 
found in its vicinity. Expressing the gene for $\beta$-galactosidase when glucose
is available or when lactose is unavailable would be a waste of energy. 
Failing to express this enzyme in the converse situation would probably
lead the cell to an "energy crisis" which could result in its death. 
Through the history of evolution by means of natural selection, genes that
code for proteins which prove to be useful in diverse contexts are conserved and
carried on across generations.
The cell has readily available "recipes" on how to react to stimuli it has 
never previously encountered, encoded in its genome. 
The dynamic selection of which ones to use at any given time is the process 
of gene regulation.                                                                           
\newpage


Having presented a brief example of a renowned operon, the following table [11]
clearly summarises different operon types. 
These are differentiated based on the repression/induction behaviour they exhibit.

| Positive inducible operon | Positive repressible operon | Negative inducible operon | Negative repressible operon |
|---------------------------|-----------------------------|---------------------------|-----------------------------|
  | An activator protein turns ON transcription in response to a stimulus/condition | An activator protein is inactivated in response to a stimulus/condition, turning transcription OFF | A repressor protein is inactivated in response to a stimulus/condition, turning transcription turns ON | A repressor protein turns transcription turns OFF in response to a stimulus/condition |


## Eucaryotic Transcription

Eukaryotic organisms can be much more complex than prokaryotes. 
This can be easily noted given that in multicellular eukaryotic organisms, 
cells undergo differentiation to acquire specific morphological and 
physiological characteristics. 
Cells from a human's femoral biceps, their liver and brain are all radically 
different. This can be observed directly via a microscope. 
This diversity relies as we have mentioned before, on gene expression regulation.

As in prokaryotes, gene regulation has a general mechanism in eukaryotes,
albeit alternatives may be used in specific contexts.
This general case is as we previously deduced, transcriptional regulation.

In order to explain eukaryotic transcription and its regulation, we must
first understand the basic structure of an eukaryotic gene which significantly
diverges from that of its prokaryotic counterpart.

![Schematic representation of an eukaryotic gene](figures/eukarya1)

Analogously to prokaryotes, eukaryotic genes contain regulatory 
sections and coding sections.
Nonetheless, the coding region of an eukaryotic gene is not the DNA equivalent
of the final mRNA transcript. 
The coding part of the gene contains interspersed coding and non-coding DNA fragments. 
The former are called **exons**, the later **introns**. 
The removal of introns to produce the final translatable mRNA product is
a form of post-transcriptional modification called _splicing_. 
This happens within the nucleus, hence the name *in*trons (for the 
fragments which remain inside).
Moving upstream along the gene shown in the previous figure, we 
find the **transcription start site**, 
where _RNA polymerase II_ (**Pol II**) binds to.
RNA polymerase II is a complex made up of 12 proteins which together carry
out transcription.
Eukaryotes have more than one type of RNA polymerase, but these are not
involved in the expression of most genes. 
_RNA polymerase I_ only transcribes ribosomal RNA (rRNA). 
_RNA polymerase III_ transcribes genes that code for ribosomal
5S rRNA, transfer RNA (tRNA) and other small RNAs.

Within the _promoter region_ we generally find two distinct types of promoters.
First, we find the _core promoter_. 
This is a sequence of DNA, about 40 bases upstream of the start site. 
It is found in all eukaryotic genes. 
Different core promoters exists, but a common example is the _TATA box_.
Afterwards we observe _upstream promoters_ which are located further away
from the coding region of the gene. 
Unlike the core promoter, these are specific. 
They bind to different activator or repressor proteins (transcription factors) 
conferring specificity to the process of gene expression regulation. 

Finally, we have the _enhancer_. 
It is found furthest away from the coding section of the gene 
(up to thousands of bases away), and is not restricted to be located upstream. 
The enhancer can also bind to different transcription factors. 
When this happens, the enhancer loops around and binds to the protein complex
found on the promoter. 
This interaction stimulates transcription of the gene, hence the name _enhancer_.


Regarding transcription factors, we can also make a distinction between types. 
_General transcription factors_ (**GTFs**) are named TFIIA, TFIIB, 
TFIID (*TATA-binding protein* **TBP**),
TFIIE, TFIIF, and TFIIH based on the protein fractions they purified in. 
They are called _general_ because together with _Pol II_, they are **necessary
and sufficient for basal transcription** of many genes. 
GTFs and Pol II then form the _preinitiation complex_ (**PIC**) by protein-protein
interactions. 
An important remark is that experimental evidence shows that these
known GTFs are insufficient to transcribe DPE[^a]-containing genes. 
In addition to this, it has been found that TBP paralogues are involved in 
distinct biological processes, such as embryonic development, differentiation 
and morphogenesis.

## Transcriptional Regulation

Gene expression is primarily controlled at the level of transcription by 
regulating which genes should be copied into RNA in any situation. 
The major control point is at transcriptional initiation when the DNA
begins to be copied into RNA. 
Many cases of regulation at transcriptional elongation also exist. 
This is thus a secondary mechanism of transcriptional regulation worth
mentioning.

## Transcription factors
DNA sequences which affect the rate of transcription act by binding 
regulatory proteins (*transcription factors* **TF**). 
These have a modular structure with different domains which have
specific functions such as DNA binding or transcriptional activation.
Transcription factors can be grouped into families with related DNA binding domains.
As previously presented in the operon regulation table, TFs can be either activators
or repressors.
Inhibitory TFs can repress transcription either indirectly by blocking the effect
of a positively acting factor or directly via the basal transcription complex.

Transcription factors can themselves be regulated by controlling their synthesis
or their activity. 
If transcription factors can be controlled by other transcription factors, we ultimately have a big transcriptional network which represents the 
"*cognitive engine*" of the cell.
**maybe say that this is the best?**

## Transciptional Networks

```{r libs, warning=FALSE, message=FALSE}
library(tidyverse)
gene.names <- read_delim(
  "data/coliInterFullNames.txt", delim=" ", 
  col_types =  cols(X1="i", X2="c", X3="_"),
  col_names = c("num", "name")
)
head(gene.names, 3)
```

```{r lac.selec, tidy=TRUE}
lac <- gene.names %>% filter(str_detect(name, "lac"))
knitr::kable(lac)
```

```{r interactions, tidy=TRUE}
interactions <- read_delim(
  "data/coliInterFullVec.txt", delim=" ", 
  col_names = c("operon_n", "tf_n", "reg_type")
)
head(interactions, 3)
```


```{r lac.find}
# Find lac interactions :
lac.tf <- interactions %>% filter(operon_n %in% lac$num | tf_n %in% lac$num)
lac.tf
```

```{r secondary, }
lac.int.2 <- interactions %>% 
  filter(operon_n %in% lac.tf$tf_n | # Get all regulations where lac's TFs are targets
             tf_n %in% lac.tf$tf_n   # Get all operons which are co-regulated 
        )                            # i.e. all whose expression is regulated by the same TFs
dim(lac.int.2)
```
```{r naming}
# Prepare the format
lac.int.2.pregraph <- lac.int.2 %>% 
  select(tf_n, operon_n, -reg_type) %>%
    left_join(gene.names, by = c("operon_n" = "num"), name = "op_name") %>%
      left_join(gene.names, by = c("tf_n" = "num"), name = "tf_name") %>%
        transmute(tf_n = name.y, operon_n = name.x)
head(lac.int.2.pregraph)
```

```{r the.graph, message=FALSE, warning=FALSE, fig.fullwidth=TRUE, fig.align="c", fig.height=10}
library(igraph)

# Create the graph 
lac.graph <- graph_from_edgelist(as.matrix(lac.int.2.pregraph), directed = T)
# Remove isolated nodes
isolated.nodes <- which(degree(lac.graph) == 0)
lac.graph.pruned <- delete.vertices(lac.graph, isolated.nodes)
plot.igraph(lac.graph.pruned, arrow.size = 1, mark.expand = 5, vertex.color = "yellow")
```

## Transcriptional Control: Chromatin Structure

So far, we have discussed gene expression regulation by means of transcriptional
control. 
This directly addresses the first presented _cognitive problem of the cell_.
By selectively transcribing the needed genes, cells can successively 
(or immediately in the case of prokaryotes) translate mRNA into proteins
generating a coherent response to external stimuli and their current metabolic state.
There is however a question that remains unanswered. 
How do eukaryotic cells differentiate during embryonic development?
Furthermore, once differentiated how do cells commit to a particular lineage?
This evidently requires additional levels of control. 
Other mechanisms are needed to explain the persistence
of the aforementioned differentiation.
This appears to be a _long-term memory_, contrary to the quasi immediate
functioning of transcriptional regulation.

These long-term changes which allow a cell to become and remain committed to a 
particular pattern of gene expression which determines its lineage involve 
alterations in the chromatin structure. 
These changes allow switching between the tightly-packed solenoid structure 
to the more open conformation. 
With these changes, active (or potentially) genes exhibit undermethylation 
on specific C residues of the DNA. 
Surprisingly there is evidence that these modifications, although stable,
are not irreversible.
A clear example is the previously mentioned cloning of Dolly the sheep.
If these changes were permanent, this experiment would have not produced 
a healthy clone. 


\begin{wrapfigure}{R}{5.5cm}
  \caption{Comparison of regulatory mechanisms}\label{wrap-fig:1}
  \includegraphics[width=5.5cm]{figures/two_mechanisms}
  \vspace{-270pt} % This removes the white box on the second page
\end{wrapfigure}

Two common regulatory mechanisms serve two separate needs. 
For an efficient response (one requiring less energy) the gene is often
regulated at the level of transcription. 
When a rapid response is needed, a precursor protein is produced in advance and
then rapidly converted to an active protein when the conditions warrant.

## Transcriptional networks, current issues, data mining, etc

## Conclusion : biologie integrative, etc.

[^a]: Downstream Promoter Element

\newpage
# Glossary

* **Gene**: The unit of inheritance, "a particulate factor that passes unchanged from generation to generation".

* **Deoxyribonucleic acid**: The chemical constituents of the gene segments, comprising nucleotides and a sugar phosphate
backbone, incorporated into a double helical structure.

* **Nucleotide**: The building blocks of the gene that specify the genetic alphabet. There are two types of nucleotide, purines and
pyrimidines.
* **Pyrimidines**: A nitrogenous base with a six membered ring eg Cytosine and Thymine.
* **Purines**: A nitrogenous base with fused 5 and 6 membered rings eg Adenine and Guanine.
* **Genotype**: The genetic composition of an individual.
* Phenotype: The physical expression of the genotype.
* **Allele**: the particular form of the gene (all genes except genes on the Y chromosome have 2 copies in humans).
* **Homozygote**: both alleles of a gene inherited from maternal and paternal are identical for the trait of a particular gene.
* **Heterozygote**: The alleles of a gene inherited from maternal and paternal are different.
* **Dominant**: the presence of one allele is sufficient to provide the phenotype of the individual.
* **Recessive**: the presence of both alleles is required to provide the phenotype of the individual.
* **Locus**: The physical position of the gene.
* **Polymorphism**: a change at the nucleotide level between the genomes of two individual at a specific locus. This may take
the form of a single base change, or a change in the number of repetitive elements at this locus. Many polymorphisms
are neutral, having no definable phenotype.
* **Eukaryotic**: referring to the genome of multicellular organisms, from yeast to humans genes are interrupted by introns.
* **Prokaryotic**: referring to the single cell organisms such as bacteria, archaebacteria etc, no introns present.

\newpage
# References
```{r generateBibliography, results="asis", echo=FALSE, warning=FALSE}
suppressMessages(require("bibtex"))
suppressMessages(require("knitcitations"))
cleanbib()
options("citation_format" = "pandoc")
read.bibtex(file = "references.bib")
``` 
